### PR TITLE
update building APK page

### PR DESCRIPTION
**Pull request by sethladd**
Closes https://github.com/flutter/flutter/issues/3307

### CODE REVIEW

**Comment by sethladd** [comment]
cc @qchong

File: building-apk.md [hunk 1]
```diff
@@ -4,17 +4,17 @@ title: Building an APK for Android
...
 
-_Note: An `apk/AndroidManifest.xml` file was already created for
-your app by `flutter create`. You can check it out if you like,
-but you don't need to modify it when getting started._
+Follow these steps to build a deployment-ready APK:
 
-First, ensure you installed the Android SDK tools
-(see "Setting up your Android device" from [Getting started](/getting-started)).
+1. cd into your application's directory
+1. Run `flutter build apk`
```
Comment (Hixie) [inline building-apk.md]: 2. to make this more readable :-)

File: building-apk.md [hunk 2]
```diff
@@ -4,17 +4,17 @@ title: Building an APK for Android
...
+The APK can be found in `MY_FLUTTER_APP/build/app.apk`.
 
-Then, inside your Flutter app, run `flutter build apk`.
+# Installing an APK onto a device
 
-The APK can be found in `$MY_FLUTTER_APP/build/app.apk`.
+Follow these steps to install your APK onto a connected Android device:
+
+1. Connect your Android device to your workstation/laptop
+1. cd into your application's directory
```
Comment (Hixie) [inline building-apk.md]: ditto

File: building-apk.md [hunk 3]
```diff
@@ -4,17 +4,17 @@ title: Building an APK for Android
 permalink: /building-apk/
 ---
 
-_Note: An `apk/AndroidManifest.xml` file was already created for
```
Comment (Hixie) [inline building-apk.md]: mentioning the manifest file seems useful, don't we want to mention this somewhere?

File: building-apk.md [hunk 4]
```diff
@@ -4,17 +4,17 @@ title: Building an APK for Android
...
 
-Then, inside your Flutter app, run `flutter build apk`.
+# Installing an APK onto a device
 
-The APK can be found in `$MY_FLUTTER_APP/build/app.apk`.
+Follow these steps to install your APK onto a connected Android device:
+
+1. Connect your Android device to your workstation/laptop
+1. cd into your application's directory
+1. Run `flutter install`
```
Comment (Hixie) [inline building-apk.md]: Maybe mention that "flutter run" builds, installs and runs the apk in one go?

**Comment by Hixie** [comment]
LGTM

File: building-apk.md (hunk 4 above)
Comment (sethladd) [inline building-apk.md]: Yeah, except that on the second run, you're not left in a state where you can just unplug your phone and go. On the second run of `flutter run`, the updated flx is put into a tmp directory (it doesn't actually update the APK).

File: building-apk.md (hunk 1 above)
Comment (sethladd) [inline building-apk.md]: What did you mean?

Comment (sethladd) [inline building-apk.md]: done

File: building-apk.md (hunk 2 above)
Comment (sethladd) [inline building-apk.md]: done

File: building-apk.md (hunk 3 above)
Comment (sethladd) [inline building-apk.md]: Added back.

File: building-apk.md (hunk 4 above)
Comment (sethladd) [inline building-apk.md]: Added.

Comment (Hixie) [inline building-apk.md]: I didn't know that, so yeah, definitely mention all that. :-)

**Pull request merged**